packages/theme/components/layout/root-layout/index.tsx

```diff
@@ -3,10 +3,15 @@ import React from 'react'
 
 import {ColorModeProvider} from '../../context/color-modes/ColorModeProvider'
 import {Theme, ThemeProps} from './Theme'
-import {PageMapItem} from 'nextra'
+import {type ConfigContextLink, ConfigContextProvider} from '../../context/useConfig'
+import type {PageMapItem} from 'nextra'
 
 type Props = {
   pageMap: PageMapItem[]
+  links?: {
```

Comment:
This is a really nice way to pass the config. 

`links` feels a bit too broad however. Can we split this up into more explicit props for `headerLinks` and `sidebarLinks`?

Reply:
Yes no prob. I implemented it this way to more closely match the underlying context, but I'll update that too to remove the `links` object